aksel.nav.no/website/sanity/schema/documents/admin/editorialStaff.tsx

```diff
@@ -0,0 +1,78 @@
+import { parseInt } from "lodash";
+import Image from "next/image";
+import { SlugValue, defineField, defineType } from "sanity";
+import { showForDevsOnly } from "../../../util";
+
+export const EditorialStaff = defineType({
+  title: "Redaksjon",
+  name: "editorial_staff",
+  type: "document",
+
+  fields: [
+    defineField({
+      title: "Navn",
+      name: "title",
+      description:
+        "Navnet til Redaksjonen, ofte et team feks (Nav.no, Aksel... o.l.), men kan også være en gruppering på interesseområder feks (Security champions, Fagtorsdagkomitéen... o.l.)",
+      type: "string",
+      validation: (Rule) => Rule.required().error("Må legge til navn"),
+    }),
+    defineField({
+      title: "Beskrivelse",
+      description: "En kort beskrivelse av redaksjonen. Maks 140 tegn",
+      name: "description",
+      type: "string",
+      validation: (Rule) => Rule.required().max(140).error("Maks 140 tegn"),
+    }),
+    defineField({
+      title: "avatar_id",
+      name: "avatar_id",
+      type: "slug",
+      hidden: showForDevsOnly(),
```

Comment:
as developers, we can manually change the ids later if things go wrong, this is actually really neat 🤩 

eg.
- we create a _special_ team named "Aksel" and want to give the avatar_id `aksel` -> `/public/avatars/aksel.svg`
- if we later add more svgs and give them weird names